extensions/amp-consent/0.1/consent-config.js

```diff
@@ -245,19 +244,23 @@ export class ConsentConfig {
   }
 
   /**
-   * Validate if story consent then no promptUiSrc
+   * Validate if story consent then no promptUISrc
    * @param {!JsonObject} config
    * @return {!JsonObject}
    */
   checkStoryConsent_(config) {
-    if (childElementByTag(this.element_, AMP_STORY_CONSENT_TAG)) {
-      userAssert(
-        !config['promptUISrc'],
-        '%s: `promptUiSrc` cannot be specified while using %s.',
-        TAG,
-        AMP_STORY_CONSENT_TAG
-      );
-    }
+    // TODO(wg-stories):
+    // We verify that configurations with an <amp-story-consent> descendant have
+    // no promptUISrc, but we don't verify that the element is present in
+    // Stories with <amp-consent>. Thus, Stories could be configured using a
+    // promptUISrc. This is likely not what we want for UX, so wg-stories
+    // should provide a migration path for these documents if they exist.
+    userAssert(
+      !config['promptUISrc'] ||
```

Comment:
Isn't this going to start triggering for all AMP (non story) documents using `promptUISrc`?